features/configuration_loading.feature

```diff
@@ -30,7 +30,14 @@ Feature: Offer different ways how to load configuration
 
   Scenario: Configuration file in working directory
     Given the smelly file 'smelly.rb'
-    And a configuration file '.reek.yml'
+    And a file named "config.reek.yml" with:
```

Comment:
Reek should not pick this up since it's not `.reek.yml`, but it does. https://github.com/troessner/reek/pull/1315#issuecomment-394142481

Reply:
I added an issue for this: #1363.

Reply:
And as proof for your hypothesis for that it now fails after I rebased against master 😆